configure.ac

```diff
@@ -430,6 +430,8 @@ MINGW=`echo $host_os | grep mingw32  2>/dev/null`
 AM_CONDITIONAL(MINGW, test ! -z "$MINGW" )
 if test ! -z "$MINGW"; then
 	WSOCKLIB="-lws2_32"
+	LDFLAGS="-no-undefined $LDFLAGS"
+	AC_DEFINE(HAVE_GETTIMEOFDAY)
```

Comment:
> Can't say I'm happy either, but the main reasoning was better cross-platform compatibility (see comments in #153). To be honest, this PR may be one of those to prove them right...

Indeed. Anyway many veteran programmers (specially Linuxers) prefer autotools instead of cmake, so this removing autotools files may bring some discomfort to them. :-/

> Does cmake work for you without any changes?

I need to test it. I have CLion installed on my system and it uses cmake as default, so I'm going to test libvncserver directly in CLion and back with feedback...